backend-test.js

```diff
@@ -13,7 +13,7 @@ const STORAGE_HISTORY_NAME="history"
 const STORAGE_TRANSACTIONS_NAME="transactions"
 
 // Compile Typescript
-exec("jest --json --outputFile=report.json",
```

Comment:
Can you revert this change back? Don't necessarily want the `--detectOpenHandles` flag